config_center/nacos/client.go

```diff
@@ -50,6 +51,15 @@ type NacosClient struct {
 	onceClose  func()
 }
 
+func init() {
+	switch runtime.GOOS {
+	case "windows":
+		logDir = "logs\\nacos\\log"
+	default:
+		logDir = "logs/nacos/log"
+	}
+}
+
```

Comment:
Inaddition to set default value for `logDir`, we should provide a config item to change this value

Reply:
done